tools/plugins/com.liferay.ide.core/src/com/liferay/ide/core/util/PropertiesUtil.java

```diff
@@ -907,4 +919,134 @@ public static void saveProperties( Properties props, File bladeSettings )
         }
     }
 
+    public static List<IFile> getDefaultLanguagePropertiesFromModuleProject( IProject project )
```

Comment:
Hey , Greg , I analysed the code about ILiferayProject and its implementations and I found the 
String getProperty( String key, String defaultValue ); 
method is just for getting theme related property , so we can reuse it for getting properties in components (configured via java annotation ) and it will be simple without adding new interface.
what is your opinion ?